pkg/reconciler/kubernetes/tektondashboard/tektondashboard.go

```diff
@@ -286,7 +288,7 @@ func (r *Reconciler) ReconcileKind(ctx context.Context, td *v1alpha1.TektonDashb
 func (r *Reconciler) updateTektonDashboardStatus(ctx context.Context, td *v1alpha1.TektonDashboard, createdIs *v1alpha1.TektonInstallerSet) error {
 	// update the td with TektonInstallerSet and releaseVersion
 	td.Status.SetTektonInstallerSet(createdIs.Name)
-	td.Status.SetVersion(r.releaseVersion)
+	td.Status.SetVersion(r.dashboardReleaseVersion)
```

Comment:
Shall we set both versions ?

I mean component as well as operator
WDYT?

Reply:
well yes but at present we didn't needed that 
for upgrade cond atleast
but I have added a TODO for metrics if we decided to go with it then we will have to add